src/main/kotlin/com/vk/admstorm/highlight/markers/admmarker/MarkerBrowser.kt

```diff
@@ -0,0 +1,9 @@
+package com.vk.admstorm.highlight.markers.admmarker
+
+import com.vk.admstorm.env.Service
+
+abstract class MarkerBrowser : IMarkerBrowser {
+    fun List<Service>.getByKey(key: String): Service? {
+        return this.firstOrNull { it.key == key }
```

Comment:
maybe `= firstOrNull { it.key == key }`?

Reply:
done